ambry-commons/src/main/java/com.github.ambry.commons/Account.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.commons;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONObject;
+
+
+/**
+ * A representation of an Ambry account. An account is an internal representation of a user.
+ */
+public class Account {
+  private final short id;
+  private final String name;
+  private final JSONObject metadata;
+  private final Map<Short, Container> idContainerMap;
+  private final Map<String, Container> nameContainerMap;
+
+  /**
+   * Constructor of Account.
+   * @param id The id of the account, which is the internal reference to the account.
+   * @param name The name of the account, which is the external reference to the account.
+   * @param metadata The metadata of the account in JSON.
+   */
+  public Account(short id, String name, JSONObject metadata) {
+    if (name == null) {
+      throw new IllegalArgumentException("accountName cannot be null");
+    }
+    this.id = id;
+    this.name = name;
+    this.metadata = metadata;
+    idContainerMap = new HashMap<>();
```

Comment:
`idContainerMap` doesn't read well. How about calling this `containerIdMap` or `containerIdToContainerMap`? The same with `nameContainerMap`.